base/init.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
-cp -r --no-clobber /tmp/home_nbuser_default/. /home/$NB_USER/
-rm -rf /tmp/home_nbuser_default
```

Comment:
Isn't `rm` still applicable?

Reply:
This was not removed because it went against permissions
cannot remove /home_nbuser_default : permission denied
This is because jovyan does not have permission on the / directory